mlflow/genai/judges/prompts/output_equivalence.py

```diff
@@ -0,0 +1,47 @@
+from mlflow.genai.prompts.utils import format_prompt
```

Comment:
It was added back during the conflict resolution. Removed this file in the feature branch